configure.ac

```diff
@@ -8,6 +8,8 @@ define(_CLIENT_VERSION_IS_RELEASE, false)
 define(_COPYRIGHT_YEAR, 2022)
 define(_COPYRIGHT_HOLDERS,[The %s developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[[Bitcoin Core and Dogecoin Core]])
+define(_CURRENT_RELEASE_DATE,[[Jan 01 1970]])
```

Comment:
Is it possible to make this something ISO-8601 compliant so that we don't have to be scared about internationalization? I.e. `1970-01-01`?

Reply:
Yeah, I like that better. This constant fits the Qt date parsing functions really well, but now that I look at it, it's smarter not to let a client detail suggest a suboptimal design.